src/Data/Functor/Alt.hs

```diff
@@ -55,7 +55,7 @@ import Data.List.NonEmpty (NonEmpty(..))
 import qualified Data.Monoid as Monoid
 import Data.Semigroup (Option(..), Semigroup(..))
 import qualified Data.Semigroup as Semigroup
-import Prelude (($),Either(..),Maybe(..),const,IO,Ord,(++),(.),either,seq,undefined)
+import Prelude (($),Either(..),Maybe(..),const,IO,Eq,Ord,(++),(.),either,seq,undefined)
```

Comment:
Move this `Eq` import to be within the `#ifdef MIN_VERSION_unordered_containers ... #endif` CPP, as it's only used there.